generator/views/funnel_analysis_view.py

```diff
@@ -0,0 +1,203 @@
+"""Class to describe a Funnel Analysis View."""
```

Comment:
Can you add a few more comments about what this is and the reasoning behind its architecture?